src/main/kotlin/nebula/plugin/bintray/BintrayExtension.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nebula.plugin.bintray
+
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.kotlin.dsl.property
+
+open class BintrayExtension(objects: ObjectFactory) {
+    val user: Property<String> = objects.property(String::class.java)
+    val apiKey: Property<String> = objects.property()
+    val apiUrl: Property<String> = objects.property()
+    val pkgName: Property<String> = objects.property()
+    val repo: Property<String> = objects.property(String::class.java)
```

Comment:
Why is the class sometimes present?

Reply:
good catch, I just want to keep the explicit type for lists because I believe it's required, made the change